server/components/project_repository.js

```diff
@@ -21,7 +21,7 @@ class ProjectRepository {
 
   getProjects(user, page = 0) {
     const client = github.createClient(user);
-    return client.repos.getReposAsync({
+    return client.repos.gmn etReposAsync({
```

Comment:
yes, weird typo after I had verified things.. I've updated the branch with the fix.